firebase-firestore/src/androidMain/kotlin/dev/gitlive/firebase/firestore/firestore.kt

```diff
@@ -39,9 +52,10 @@ public val LocalCacheSettings.android: com.google.firebase.firestore.LocalCacheS
     }
 }
 
-public actual typealias NativeFirebaseFirestore = com.google.firebase.firestore.FirebaseFirestore
+internal actual typealias NativeFirebaseFirestore = AndroidFirebaseFirestore
 
-public val FirebaseFirestore.android: NativeFirebaseFirestore get() = native
+public operator fun FirebaseFirestore.Companion.invoke(android: AndroidFirebaseFirestore): FirebaseFirestore = FirebaseFirestore(android)
```

Comment:
With the public construcor gone (see https://github.com/GitLiveApp/firebase-kotlin-sdk/pull/555/files#r1655371860), in order to still support `FirebaseFirestore(android: AndroidFirebaseFirestore)` we can add an extension to the companion object https://github.com/GitLiveApp/firebase-kotlin-sdk/pull/555/files#r1655372358

This maintains the same syntax, but since it is an extension method, it does require an explicit import